plugin/storage/cassandra/factory.go

```diff
@@ -131,6 +137,262 @@ func (f *Factory) configureFromOptions(o *Options) {
 	}
 }
 
+const (
+	MODE               = `MODE`
+	DATACENTER         = `DATACENTER`
+	TRACE_TTL          = `TRACE_TTL`
+	DEPENDENCIES_TTL   = `DEPENDENCIES_TTL`
+	KEYSPACE           = `KEYSPACE`
+	REPLICATION_FACTOR = `REPLICATION_FACTOR`
+	VERSION            = `VERSION`
+	COMPACTION_WINDOW  = `COMPACTION_WINDOW`
+)
+
+// Parameters required for initilizing the db
+type StorageConfigParams struct {
```

Comment:
@yurishkuro @akstron i'm working on a proposal to redefine the configs for cassandra in https://github.com/jaegertracing/jaeger/issues/5928#issuecomment-2334951768. What are both of your thoughts on adding the configurations here into a `mapstructure:"schema"` grouping?

Reply:
Do you think schema and data are different?

Reply:
Sorry I should've mentioned. I was suggesting consolidating data and schema into one.

Reply:
Ok makes sense

Reply:
Looks good to me.